第一周任务修改

ZhangNan13 opened the pull request:
对第一周的任务进行了修改，此部分中实现了Hash值的计算以及key-value的存储。
Hash值的计算参考课上提出的问题修改了原有代码。
在存储部分，将单个key-value以文件形式存储在规定路径E:\\MyGit\\object中。存储前先寻找是否存在文件，若存在key值为文件名的文件，不会重复存储。
经过测试，能成功计算文件与文件夹的Hash值并对key-value进行存储。
@wxc1217 @RitaPeng0101

ZhangNan13 committed:
第一周任务修改

对第一周的任务进行了修改，此部分中实现了Hash值的计算以及key-value的存储。
Hash值的计算参考课上提出的问题修改了原有代码。
在存储部分，将单个key-value以文件形式存储在规定路径E:\\MyGit\\object中。存储前先寻找是否存在文件，若存在key值为文件名的文件，不会重复存储。
经过测试，能成功计算文件与文件夹的Hash值并对key-value进行存储。
@wxc1217 @RitaPeng0101

wxc1217 commented on Hash.java:
```diff
@@ -0,0 +1,165 @@
+    }
+
+    /* 对当前文件夹进行遍历，储存文件及文件夹的key-value值 */
+    public static String Treehash(String path) throws IOException {
```
这里的函数名是不是应该改成Show_KVstore之类的，函数中是实现了键值对的存储

wxc1217 replied:
我觉得是不是要把键值对的存储和展示分开，这样之后在新增文件或其他需要重新存储键值对的情况下，就不用每次都输出一遍现有的键值对合集了。

ZhangNan13 replied:
我理解的是这个代码求出并保存了文件夹的hash值，要改也可以。

ZhangNan13 replied:
这个代码应该每次只会输出更改后的键值对，没有更改则hash值不变，不会输出。

wxc1217 commented on Hash.java:
```diff
@@ -0,0 +1,165 @@
+    }
+
+    /* 对当前文件夹进行遍历，储存文件及文件夹的key-value值 */
+    public static String Treehash(String path) throws IOException {
```
我觉得是不是要把键值对的存储和展示分开，这样之后在新增文件或其他需要重新存储键值对的情况下，就不用每次都输出一遍现有的键值对合集了。

wxc1217 reviewed (commented):
这里的函数名是不是应该改成Show_KVstore之类的，函数中是实现了键值对的存储

ZhangNan13 committed:
方法名称修改

依照大家的建议，在原来的基础上修改了方法名，深度遍历并存储Hash值的方法名称修改为Show_KVstore，求文件夹Hash值的方法名称修改为Treehash。
@wxc1217 @RitaPeng0101

ZhangNan13 committed:
Hash修改

1.对文件存储及读取方法进行了修正，将路径变量添加为输入值，增加了文件存储及读取方法的可复用性。
2.新增了initsetting方法，用于创建存储git内容的文件夹，解决文件存储时无法找到路径的问题。
